docs/unified/novacustom/features.md

```diff
@@ -0,0 +1,140 @@
+# Special features
+
+This document describes special firmware features specific to NovaCustom
+laptops.
+
+## Battery charge management
+
+NovaCustom laptops equipped with Dasharo firmware provide several battery
+management mechanisms to optimize user experience and further prolong the
+battery's lifespan.
+
+### Low battery boot prevention
+
+To prevent corruption of the bootloader, which may occur due to sudden power cut
```

Comment:
```suggestion
To prevent corruption of the bootloader, which may occur due to a sudden power cut
```

Probably will need a spellchek/grammarly to be applied.

Reply:
Applied in https://github.com/Dasharo/docs/pull/807/commits/982cef4ff5b863e6c0ca4fedde4ec0938cbc086e

Reply:
Grammarly did not complain anymore?

Reply:
Indeed it did. Applied more fixes in https://github.com/Dasharo/docs/pull/807/commits/a904f27b63611ad6b7a8ea12e1c69a8d2f5319ff